pagination/src/main/java/com/cerner/beadledom/pagination/parameters/LimitParameter.java

```diff
@@ -68,10 +59,18 @@ protected Integer parse(String param) throws InvalidParameterException {
               + " - int is required.");
     }
 
-    if (limit < 0 || limit > maxLimit) {
```

Comment:
Would something like this be more readable?
```java
int minlimit = (offsetPaginationConfiguration.allowZeroLimit()? 0:1;
if ( limit < minLimit)
        || limit > offsetPaginationConfiguration.maxLimit()) {
      throw InvalidParameterException.create(
          "Invalid value for '" + this.getParameterFieldName() + "': " + this.limit
              + "  - value between "+minlimit+" and " + offsetPaginationConfiguration.maxLimit()
              + " is required.");
    }
```

Reply:
I think it does. I'm going to do that and extract it into a private method as well.

Reply:
https://github.com/cerner/beadledom/pull/118/commits/a3acf4d6055c5c2ad2a61dcd75c4fe21077d12bf